x-pack/plugins/alerting/server/rules_client/lib/validate_rule_aggregation_fields.ts

```diff
@@ -18,6 +18,7 @@ const ALLOW_FIELDS = [
   'alert.attributes.alertTypeId',
   'alert.attributes.enabled',
   'alert.attributes.params.*',
+  'alert.attributes.params.immutable', // TODO: Add support for "*" to validateFields() and remove this element
```

Comment:
is this comment ment to be left here?  If so, we'll need some more direction on what we're supposed to do here in the future ...

Reply:
@pmuellr So this comment relates to the line above: `alert.attributes.params.*` does not add support for aggregations over any params. This is why I had to explicitly add the `alert.attributes.params.immutable` parameter to the list. The comment says that the `validateFields` function doesn't take into account `*` in the field pattern, and support for that should be added to the function. Let me know how we can make the comment more clear.

Reply:
@pmuellr Thanks, this is a good idea. I created the ticket:
https://github.com/elastic/kibana/issues/159602